code/modules/antagonists/event/datum_eventmisc.dm

```diff
@@ -0,0 +1,15 @@
+RESTRICT_TYPE(/datum/antagonist/event_misc)
```

Comment:
the spelling of "event misc" shows up in some cases as `event_misc` (the type name) but `eventmisc` in other cases (filename, SSticker var). Could you choose one spelling and make these consistent?

Reply:
Yeah some of that is just copy over from older code. I will go ahead and change it so it uses snake case, which is what I prefer given our styling guidelines.

Reply:
Went back to using eventmisc to keep things easier.